.github/workflows/run-forge.yaml

```diff
@@ -72,54 +67,41 @@ env:
   FORGE_NAMESPACE: ${{ inputs.FORGE_NAMESPACE }}
   FORGE_TEST_SUITE: ${{ inputs.FORGE_TEST_SUITE }}
   POST_TO_SLACK: ${{ inputs.POST_TO_SLACK }}
-  USE_NEW_WRAPPER: ${{ inputs.USE_NEW_WRAPPER }}
   FORGE_ENABLE_FAILPOINTS: ${{ inputs.FORGE_ENABLE_FAILPOINTS }}
 
 jobs:
   forge:
+    if: env.FORGE_ENABLED == 'true'
```

Comment:
does this actually work? Not sure if `env` context is available at the job level

Reply:
I can just revert all this then

Reply:
Opting to just remove this entirely, if we need to disable we can just use the magic of pull request target or modify the branch protection rules